Assignment/Program.cs

```diff
@@ -1,7 +1,11 @@
-﻿namespace Assignment {
-   internal class Program {
-      static void Main (string[] args) {
-         Console.WriteLine ("Hello world!");
-      }
-   }
+﻿Console.Write ("Enter a value : ");

```

Comment:
Please add comments.

Reply:
Also please try to add the basic version of old school swap of numbers. (i.e., without C sharp style like (a,b) = (b,a))

Reply:
Done.